packages/patternfly-3/react-console/src/VncConsole/__snapshots__/VncActions.test.js.snap

```diff
@@ -1,43 +1,65 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`VncActions renders correctly component hierarchy 1`] = `
-<DropdownButton
-  bsStyle="default"
-  id="console-send-shortcut"
-  onClick={[Function]}
-  title="My Send Shortcut description"
->
-  <MenuItem
-    bsClass="dropdown"
+<div>
```

Comment:
additional div?

Reply:
No, `div` should be there. https://github.com/patternfly/patternfly-react/blob/223e582264d215dd6341bd87507a2774b4b0e4b0/packages/patternfly-3/react-console/src/VncConsole/VncActions.js#L17-L28